docs/tutorials/FlowTutorial.md

```diff
@@ -78,8 +76,8 @@ follows:
 export OS_NAME=centos7
 docker run --rm -it -u $(id -u ${USER}):$(id -g ${USER}) -v $(pwd)/flow:/OpenROAD-flow-scripts/flow openroad/flow-$OS_NAME-builder
 ```
-**Optional:** To launch OpenROAD gui inside the docker, use command
-from [here](https://openroad-flow-scripts.readthedocs.io/en/latest/user/BuildWithDocker.html#enable-gui-support).
+> **Optional:** To launch OpenROAD gui inside the docker, use command
```

Comment:
Does this work for all OS cases without exception?

Reply:
yes

Reply:
For macOS users:
Follow this tutorial: https://cntnr.io/running-guis-with-docker-on-mac-os-x-a14df6a76efc

Then use:
docker run --rm -it -e DISPLAY=<IP_LIKE_FROM_TUTORIAL>:0 --network host --privileged <IMAGE_NAME>

For Windows I have no way to test.

Reply:
We are not yet fully supporting MacOS installation. It's always default to Centos/Ubuntu/Debian OS only.
I'll update the Docker gui support based on OS name as per your update.

Reply:
This is not a macOS installation, is using Docker in a macOS system. Building and using Docker is supported, maybe add a note that we only test in x86 systems, but is supported.